spec/integration/api/v1/vm_hostname_spec.rb

```diff
@@ -146,6 +148,19 @@ def app()
           expect(vm['lifetime']).to be_nil
         end
 
+#       it 'does not allow a lifetime to be extended past config 168' do
+#         app.settings.set :config,
+#                          { :config => { 'max_lifetime_upper_limit' => 168 } }
+#         create_vm('testhost', redis)
+#
+#         set_vm_data('testhost', "checkout", (Time.now - (69*60*60)), redis)
+#         puts redis.hget("vmpooler__vm__testhost", 'checkout')
+#         put "#{prefix}/vm/testhost", '{"lifetime":"100"}'
+#         expect_json(ok = false, http = 400)
+#
+#         vm = fetch_vm('testhost')
+#         expect(vm['lifetime']).to be_nil
+#       end
```

Comment:
What's with this being commented out vs removed?

Reply:
> It looks like a ton of this is variations on a theme... it felt like I was seeing the same change over and over. I assume this is just from where you were swapping out the redis stuff... is that accurate?

Yes. Implementing the redis connection pool required a ton of changes, so a large portion of the changes are changing how we are calling redis so we are utilizing the connection pool.

Reply:
> What's with this being commented out vs removed?

That is a failed rebase on my part. I brought in a test that was removed. I had commented it out while I could not figure out why it was not working until I later realized that it was a rebase conflict that I fixed that brought it back, after a commit had removed the test along with a change. I have a change coming that will remove this.